### PR TITLE
ssr: Add hidden auth

### PR DESCRIPTION
**Pull request by backspace**
This adds an opt-in button that links to the existing `scheme-auth` route. This is an MMMVP with scant design meant to facilitate authentication so we can proceed with the editing UI that only the owner of a Card Space should see and be able to exercise. The opting-in works by having an `edit` query parameter as well as being shown by default on preview deployments.

When auth is present, all that shows is “Authed for this space”.

csauthbutton

This is based on #2834 for now so it gets preview deployments. It also contains 5682f75 as a fix for a race condition between the initialisations of `hub-authentication` and `layer2-network` services.

### CODE REVIEW

File: packages/ssr-web/tests/acceptance/visit-card-space-test.ts [hunk 1]
```diff
@@ -191,6 +209,56 @@ module('Acceptance | visit card space', function (hooks) {
...
+          await visit('/');
+
+          assert.dom('[data-test-connect-button]').doesNotExist();
+          assert.dom('[data-test-auth-for-this-space]').exists();
+        });
+      }
+    );
+
+    module(
+      'authed for another space',
```
Comment (Aierie) [inline packages/ssr-web/tests/acceptance/visit-card-space-test.ts:240]: this is more "has auth token but is not owner" I think.

**Review by Aierie — APPROVED** [review]

**Comment by Aierie** [comment]
Testing this i realise it's good to set the space Meta tags and the WalletConnect site info to reflect the owner of the space.

**Comment by Aierie** [comment]
Also, since the primary purpose of auth is to allow editing, what do you think of changing the query param to "edit-mode" from "auth"?

**Comment by backspace** [comment]
I’m puzzled why it’s showing 101 commits… but I’m going to wait for #2852 to be merged and then try to understand, maybe that’s related.

File: packages/ssr-web/app/components/card-space/user-page/index.hbs [hunk 2]
```diff
@@ -30,6 +31,30 @@
...
+      {{#if this.hubAuthentication.showAuth}}
+        <section class='card-space-user-page__auth'>
+          {{#if this.hubAuthentication.isInitializing}}
+            Loading…
+          {{else}}
+            {{#if this.hubAuthentication.isAuthenticated}}
+              {{#if this.authIsForModel}}
+                <span data-test-auth-for-this-space>Authed for this space</span>
+              {{else}}
+                <span data-test-auth-not-for-this-space>Authed but not for this space</span>
```
[the remark this answers is not in the document]
Comment (backspace) [reply packages/ssr-web/app/components/card-space/user-page/index.hbs:44]: this is a good point, the two different types of auth… I’m going to declare that out of scope for this but I think we should definitely revisit and refine, it should be possible to communicate these nuances somehow, and as we discussed, maybe we don’t care about not having a Hub token unless someone truly wants to edit

**Comment by backspace** [comment]
> Also, since the primary purpose of auth is to allow editing, what do you think of changing the query param to "edit-mode" from "auth"?

I changed to `edit`… shorter 😆

**Pull request merged**